examples/Console/TestLogs.cs

```diff
@@ -58,28 +58,29 @@ internal static object Run(LogsOptions options)
                         // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
                         AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
 
+                        var protocol = OpenTelemetry.Exporter.OtlpExportProtocol.Grpc;
+
                         if (options.Protocol.Trim().ToLower().Equals("grpc"))
                         {
-                            opt.AddOtlpExporter(otlpOptions =>
-                            {
-                                otlpOptions.Protocol = OpenTelemetry.Exporter.OtlpExportProtocol.Grpc;
-                            });
+                            protocol = OpenTelemetry.Exporter.OtlpExportProtocol.Grpc;
                         }
                         else if (options.Protocol.Trim().ToLower().Equals("http/protobuf"))
                         {
-                            opt.AddOtlpExporter(otlpOptions =>
-                            {
-                                otlpOptions.Protocol = OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
-                            });
+                            protocol = OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
                         }
                         else
                         {
                             System.Console.WriteLine($"Export protocol {options.Protocol} is not supported. Default protocol 'grpc' will be used.");
-                            opt.AddOtlpExporter(otlpOptions =>
-                            {
-                                otlpOptions.Protocol = OpenTelemetry.Exporter.OtlpExportProtocol.Grpc;
-                            });
                         }
+
+                        opt.AddOtlpExporter(otlpOptions =>
+                        {
+                            otlpOptions.Protocol = protocol;
+                            if (!string.IsNullOrWhiteSpace(options.Endpoint))
+                            {
+                                otlpOptions.Endpoint = new Uri(options.Endpoint);
+                            }
```

Comment:
Hmm, I guess TestMetrics.cs is also missing this functionality. At a minimum, probably makes sense to add it there too. Though, seems like there may be an opportunity to share some code between these examples, but maybe makes sense to consider that separately.

Reply:
Added the same `Endpoint` command line option to metrics example. I did some minor cleanup to remove dup code, but agree that further cleanup could be done separately.